packages/client/hmi-client/src/services/model.ts

```diff
@@ -175,24 +175,6 @@ export function getModelType(model: Model | null | undefined): AMRSchemaNames {
 	return AMRSchemaNames.PETRINET;
 }
 
-// Converts a model into LaTeX equation, either one of PetriNet, StockN'Flow, or RegNet;
-export async function getModelEquation(model: Model): Promise<string> {
-	const unSupportedFormats = ['decapodes'];
-	if (unSupportedFormats.includes(model.header.schema_name as string)) {
-		console.warn(`getModelEquation: ${model.header.schema_name} not supported `);
-		return '';
-	}
-
-	/* TODO - Replace the GET with the POST when the backend is ready,
-	 *        see PR https://github.com/DARPA-ASKEM/sciml-service/pull/167
-	 */
-	const response = await API.get(`/transforms/model-to-latex/${model.id}`);
-	// const response = await API.post(`/transforms/model-to-latex/`, model);
-	const latex = response?.data?.latex;
-	if (!latex) return '';
-	return latex ?? '';
-}
```

Comment:
Note @Tom-Szendrey until we have a replacement for `model-to-latex` functionality, which maybe @YohannParis is evaluating, this is "clean up SciML", not "remove SciML".

Reply:
I was just double checking this actually! 
Good to know I will add this and its requirements back

Reply:
See https://github.com/DARPA-ASKEM/terarium/issues/4090

Reply:
ahh perfect! Thanks Dan